tests/e2e/tests/critical-path/browser/search-capabilities.e2e.ts

```diff
@@ -234,38 +239,84 @@ test
         await common.reloadPage();
         await t.expect(browserPage.keyListTable.textContent).contains(notSelectedIndexText, 'Search by Values of Keys section not opened');
     });
+
 test
-    .before(async() => {
-        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneBigConfig, ossStandaloneBigConfig.databaseName);
-        indexName = common.generateWord(10);
-        const command = `FT.CREATE ${indexName} ON hash SCHEMA "" text`;
-        await cliPage.sendCommandInCli(command);
+    .only
+    .before(async () => {
+        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneBigConfig, bigDbName);
+        await addNewStandaloneDatabasesApi(databasesForAdding);
```

Comment:
if you add only 1 database, it's better to use 'addNewStandaloneDatabaseApi(ossStandaloneConfig)' method here and delete 'databasesForAdding' variable above